linode/manager
# change: [M3-7561] - UX & Copy Revisions: Monthly Network Transfer Pool Modal

## Description 📝
This PR updates UX and copy for Monthly Network Transfer Pool Modal.


## Target release date 🗓️
8/05

## Preview 📷
| Before  | After   |
| ------- | ------- |
|![image](https://github.com/user-attachments/assets/b641a898-3223-4c45-b255-d14e905ffc1c) |![image](https://github.com/user-attachments/assets/2ff8fe8c-d754-42a9-8c95-995135a401c0)|

## How to test 🧪


### Verification steps
(How to verify changes)
- Navigate to linodes landing page and click on Monthly Network Transfer Pool.
- Verify UX and copy changes as mentioned in the ticket.

## As an Author I have considered 🤔

*Check all that apply*
